addons/website_slides/controllers/main.py

```diff
@@ -422,23 +432,25 @@ def slides_channel_all_values(self, slide_category=None, slug_tags=None, my=Fals
            with certifications;
           :param string slug_tags: if provided, filter the slide.channels having
             the tag(s) (in comma separated slugified form);
-          :param bool my: if provided, filter the slide.channels for which the
-           current user is a member of
+          :param bool my: if provided, display slide.channels for which the
+            current user is a member of (doesn't impact course filtering)
           :param dict post: post parameters, including
+          :param int|None page: The current page number. Set to None to disable pagination (default).
+          :param int page_size: number of element per page
 
            * ``search``: filter on course description / name;
         """
         options = self._get_slide_channel_search_options(
-            my=my,
```

Comment:
Can't we remove it from _get_slide_channel_search_options parameters also ? It seems useless anyway as it is used for "search with fuzzy" ?